### ISSUE TITLE
Accessibility enhancement

### ISSUE THREAD

**Issue by Yulianav09** [Yulianav09/Portfolio#10]
# Accessibility enhancement
## Page titles
Review the title on the page.

- [x] Change the title in the `<head>` https://github.com/Yulianav09/Portfolio/blob/b15506c650963a54559d043754bcb047972c35ab/index.html#L8

## Image text alternatives** `<img alt="alt text">`
Reviewed the alt text of all images in the project.

- [x] All the images have alt text

- [x] Change the alt text for the images for the works section to be more descriptive 

## Text headings
Reviewed all the text headings in the project.

Text headings - no issues found

## Color contrast
Reviewed the color contrast in the project.

Color contrast - no issues found

## Resize
Reviewed the resized text in the project.

Resize - no issues found

## Interaction (labels are optional but recommended as voiceover tools can read them, and should be set as invisible to match the ## ## Figma design)
Reviewed the interaction in the project.

Interaction - no issues found

## Moving content
Reviewed the moving content in the project.

Moving content - no issues found

## Multimedia
Reviewed the multimedia in the project.

Multimedia - no issues found

## The basic structure of the page
Reviewed the basic structure of the page in the project.

 The basic structure of the page - no issues found
